packages/app-desktop/commands/exportDeletionLog.ts

```diff
@@ -0,0 +1,54 @@
+
+import { CommandRuntime, CommandDeclaration } from '@joplin/lib/services/CommandService';
+import { _ } from '@joplin/lib/locale';
+import Logger from '@joplin/utils/Logger';
+import shim from '@joplin/lib/shim';
+import Setting from '@joplin/lib/models/Setting';
+import bridge from '../services/bridge';
+
+export const declaration: CommandDeclaration = {
+	name: 'exportDeletionLog',
+	label: () => _('Export deletion log'),
+};
+
+const getDeletionLines = async (filePath: string) => {
+	const logFile: string = await shim.fsDriver().readFile(`${Setting.value('profileDir')}/${filePath}`);
+
+	const deletionLines = logFile
+		.split('\n')
+		.filter(line => line.includes('DeleteAction'));
+
+	if (!deletionLines.length) return '';
+
+	return `${deletionLines.join('\n')}\n`;
+};
+
+export const runtime = (): CommandRuntime => {
+	return {
+		execute: async () => {
+			if (!Logger.globalLogger) {
+				throw new Error('Could not find global logger');
+			}
```

Comment:
Again, why do you need this check for? You don't use the logger anywhere in that code below.